### PR TITLE
update hero links

### PR DESCRIPTION
**Pull request by Dhaulagiri**
### :pushpin: Summary

Update the "hero" links on the Helios home page to be more useful for where we're at with Helios at the moment.

### :camera_flash: Screenshots

Screenshot 2024-05-03 at 05 24 01

### :link: External links

Jira ticket: [HDS-3299](https://hashicorp.atlassian.net/browse/HDS-3299)

***

[HDS-3299]: https://hashicorp.atlassian.net/browse/HDS-3299?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: website/app/templates/index.hbs [hunk 1]
```diff
@@ -11,16 +11,21 @@
         consistent, accessible, and delightful product experiences.</p>
       <div class="doc-page-home__hero-links">
         <Doc::LinkWithIcon
-          @route="show"
-          @model="getting-started/for-designers"
-          @label="Start designing with Helios"
+          @href="https://go.hashi.co/hds-rollout"
+          @label="Helios Roadmap"
           @icon="arrow-right"
```
Comment (heatherlarsen) [inline website/app/templates/index.hbs:16]:
```suggestion
          @icon="external-link"
```

File: website/app/templates/index.hbs [hunk 2]
```diff
@@ -11,16 +11,21 @@
...
-          @label="Install Helios in your project"
+          @model="whats-new/release-notes"
+          @label="Release Notes"
+          @icon="arrow-right"
+          @isAnimated={{true}}
+        />
+        <Doc::LinkWithIcon
+          @href="https://go.hashi.co/hds-feedback"
+          @label="Share Feedback"
           @icon="arrow-right"
```
Comment (heatherlarsen) [inline website/app/templates/index.hbs:29]:
```suggestion
          @icon="external-link"
```

**Review by heatherlarsen — COMMENTED** [review]
I recommend changing the external links icons to `external-link` so it reflects what will happen when the user clicks on it. Because of this, it makes sense to put the Release Notes first so the internal link is first. wdyt?

**Comment by Dhaulagiri** [comment]
@heatherlarsen good call. Have a look now and see if you like that arrangement better.

**Pull request merged**